fms_extras/utils/cache/paged.py

```diff
@@ -0,0 +1,1044 @@
+import abc
+import dataclasses
+import queue
+from typing import Dict, List, Optional, Tuple, Union
+
+import torch
+import torch._inductor.ir as ir
+import torch._inductor.lowering as lowering
+from torch._dynamo import mark_static_address
+from torch._inductor.virtualized import V
+
+from fms_extras.paged_c import attn_ops, cache_ops  # type: ignore
+from fms_extras.utils.cache import (
+    AttentionComputationMixin,
+    CacheDataLayer,
+    CacheDataWithMetadata,
+    KVCache,
+    KVCacheManager,
+)
+
+
+lib = torch.library.Library("paged_attention", "FRAGMENT")
```

Comment:
what's the purpose of putting our operators in the torch namespace? I'm assuming it's something to do with making this compile, but we might want a comment explaining the motive

Reply:
yes, that is the reason, ill add a comment